templates/order/order-details.php

```diff
@@ -19,7 +19,7 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
-if ( ! $order = wc_get_order( $order_id ) ) {
+if ( ! $order = wc_get_order( $order->get_id() ) ) {
```

Comment:
There is no `$order` variable before this line of code.
This will throw an undefined variable PHP notice...